index.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const login = require('../controllers/index');
+const proxyAPI = require(libShared + '/lib/proxy-api');
+
+module.exports = app => {
+	app.post('/post/testAjax', (req, res) => res.status(204).end());
+	app.get('/change-tenant', login.changeTenant);
+	app.get('/', login.controller);
+	app.get('/change-tenant', login.changeTenant);
+	app.get('/user', (req, res, next) => {
+		if (req.session && req.session.agent) {
+			res.json({
+				id: req.session.agent.userId,
+				agentId: req.session.agent.id,
+				firstName: req.session.agent.firstName,
+				lastName: req.session.agent.lastName,
+				email: req.session.agent.userEmail,
+				scopes: req.session.agent.scopes,
```

Comment:
our standard is in sigular, please change scopes to scope

Reply:
Ok you are right